### PR TITLE
User Base Points queried

### PR DESCRIPTION
**Pull request by puneeth2001**

### CODE REVIEW

File: algorithms/repository_rating.py [hunk 1]
```diff
@@ -415,7 +415,7 @@ def calcRepoScore(self):
 
 
     
-accessToken = "03e5d817f468829fd9b3307f55de055461460c1a"
+accessToken = "<token-here>"
 headers = {"Authorization": "bearer "+ accessToken }
```
Comment (aswinshenoy) [inline algorithms/repository_rating.py]: would have liked if consumed it from SETTINGS (assuming we have a server_settings.py with the token), or implementing Github Auth and getting the token.

Comment (aswinshenoy) [reply algorithms/repository_rating.py]: This should be done in the subsequent versions perhaps.

**Comment by vchrombie** [comment]
@puneeth2001, be careful while pushing the code. Make sure you remove the tokens before sending a PR.

@aswinshenoy, you can merge this. :+1:

File: user/templates/user/base.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+{%load static%}
+<html>
+    <head>
+        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
+        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
```
Comment (aswinshenoy) [inline user/templates/user/base.html]: Use Bootstrap 4, bootstrap 3 is obselete.

File: algorithms/repository_rating.py [hunk 3]
```diff
@@ -413,19 +417,19 @@ def calcMeritScore(self):
...
 
 print()
 print("The Base Points: " + str(r.basePoints))
 print("The Activity Points: " + str(r.activityPoints))
 print("The Inclusivity Points: " + str(r.inclusivityPoints))
 print("The Merit Points: " + str(r.meritPoints))
 print()
-print("The Repository Score is " + str(r.repoPoints))
\ No newline at end of file
+print("The Repository Score is " + str(r.repoPoints))
```
Comment (aswinshenoy) [inline algorithms/repository_rating.py]: Comment these outputs as they wouldn't be required now as we have integrated it with Django.

**Pull request merged**